src/htmlrenderer.cpp

```diff
@@ -371,12 +374,21 @@ void HtmlRenderer::render(std::istream& input,
 						tables.size() ? 0
 						: indent_level);
 				}
-				inside_li = true;
+				list_elements_stack.push_back(HtmlTag::LI);
 				add_nonempty_line(curline, tables, lines);
 				prepare_new_line(curline,
 					tables.size() ? 0 : indent_level);
 				indent_level += 2;
-				if (is_ol && ol_counts.size() != 0) {
+
+				auto latest_list = std::find_if(list_elements_stack.rbegin(),
```

Comment:
Can this be marked `const`?

(If you see a tool that can do the const-correctness checks, please tell me. Clang-tidy, which seems to be the most advanced linter, can't do that :( Google didn't turn anything up, either.)

Reply:
Jup, done.

> If you see a tool that can do the const-correctness checks, please tell me.

I'm not aware of any such tool.